cbits/fpstring.c

```diff
@@ -82,13 +82,34 @@ unsigned char fps_maximum(unsigned char *p, size_t len) {
 }
 
 /* find minimum char in a packed string */
+#if defined(__x86_64__)
+unsigned char fps_minimum(unsigned char *p, size_t len) {
+    unsigned char *q, c = *p;
+    q = p;
+    if (len > 16) {
```

Comment:
How about initializing `mins` full of `FF` bytes? That way you don't need `if (len > 16)` and eliminate one level of nesting.

Reply:
Good point! It definitely would make the code cleaner, but as you said I'm not sure its worth the performance slip on small  bytestrings.